GitUI/UserControls/FileStatusList.cs

```diff
@@ -108,6 +108,26 @@ Bitmap ScaleHeight(Bitmap input)
             }
         }
 
+        private void FileStatusListView_ClientSizeChanged(object sender, EventArgs e)
+        {
+            if (!FileStatusListView.IsHandleCreated)
+            {
+                return;
+            }
+
+            var formatter = new PathFormatter(FileStatusListView.CreateGraphics(), FileStatusListView.Font);
```

Comment:
Can this be created in .ctor and then reused?
Curious whether `FileStatusListView.CreateGraphics() == eventArgs.Graphics`....

Reply:
No, because Graphics must correspond to correct Control.Handle which as far as I know is not immutable.

Reply:
When does a handle change?  https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.control.handlecreated

Reply:
When some of `ListView` settings change. [example](https://referencesource.microsoft.com/#System.Windows.Forms/winforms/Managed/System/WinForms/ListView.cs,976). You may also find other usages of `RecreateHandleInternal()` there.